app/controllers/early_allocations_controller.rb

```diff
@@ -5,44 +5,48 @@ class EarlyAllocationsController < PrisonsApplicationController
 
   def new
     case_info = CaseInformation.find_by offender_id_from_url
-    @early_assignment = case_info.early_allocations.new
+    @early_assignment = EarlyAllocationStage1.new offender_id_from_url
```

Comment:
I'm not sure why we've used the name `@early_assignment` throughout the EA controller/views, but it's a bit confusing because nowhere else do we call it an 'assignment'. Perhaps this is an old name which is no longer used?

It would be nice to clean this up and use the name `@early_allocation` – or even `@assessment` as it seems to be colloquially known.